packages/algob/test/fixture-projects/support-external-parameters/assets/stateful.py

```diff
@@ -1,5 +1,6 @@
 # This example is provided for informational purposes only
 import sys
+sys.path.insert(0,'.')
 from algobpy.parse import parse_params
```

Comment:
```suggestion
from .algobpy.parse import parse_params
```

Reply:
we should update the scripts below as well.

Reply:
After applying suggestion i get the error:
![image](https://user-images.githubusercontent.com/33264364/119791801-c87e1000-bef2-11eb-8432-d6d97eee5bc7.png)

Reply:
Can you print `os.getcwd()` and see what's the current directory and import relative path accordingly?

Reply:
I wasn't able to log `cwd` in the python script (after removing `stub` in `test/setup.js` as well). Was reading [https://techwithtech.com/importerror-attempted-relative-import-with-no-known-parent-package/](https://techwithtech.com/importerror-attempted-relative-import-with-no-known-parent-package/), but the methods here don't work, and i was getting the same error as shown above.
I will read more about it during the weekend, for now i have moved `algobpy` to `/assets` and removed  `sys.path.insert(0,'.')`.